core/capabilities/gateway_connector/service_wrapper.go

```diff
@@ -0,0 +1,164 @@
+package gatewayConnector
```

Comment:
I believe the convention is to use gateway_connector here.

Reply:
lint barfed and said no underscores in package names.

Reply:
Typically Go package names are not even full words, let alone multiple words.

> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps. They are often simple nouns...

Some resources:
- https://go.dev/blog/package-names
- https://go.dev/doc/effective_go#package-names

Reply:
In this case, perhaps something like `gatewayconn`.

We should keep the directory aligned too, rather than just rename the package.

Reply:
Interesting... we have a lot of package names with underscores throughout core. I'll remember for the future, thanks!

Reply:
I find this guidance a bit confusing. gatewayConnector is bad but gatewayconn is good?  I find the 2nd a bit harder to read because it is 2 words and almost all the same height.